src/agent/docker-rest-agent/server.py

```diff
@@ -43,7 +43,8 @@ def create_node():
     port_map = ast.literal_eval(request.form.get("port_map"))
     volumes = [        
         '/var/run/:/host/var/run/', 
-        '/opt/fabric/{}/msp:/etc/hyperledger/fabric/msp'.format(node_name),
+        '/opt/fabric/{}:/etc/hyperledger/fabric'.format(node_name),
```

Comment:
Suggest put two local paths into the same cello dir, e..g,

/opt/cello/hyperledger/fabric/{}:/etc/hyperledger/fabric
/opt/cello/hyperledger/production/{}:/var/hyperledger/production